package.json

```diff
@@ -14,9 +15,7 @@
     "url": "git+https://github.com/publiclab/leaflet-environmental-layers.git"
   },
   "keywords": [],
-  "author": {
-    "name": "Sagarpreet Chadha"
-  },
+  "author": "Sagarpreet Chadha",
```

Comment:
Let's remove these changes from here as it is not part of this issue. Please make a new issue for this. Thanks!